components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-overview.tsx

```diff
@@ -475,6 +498,16 @@ export const PortfolioOverview = ({ onToggleShowIpfsBanner }: Props) => {
                 }
                 token={item.asset}
                 hideBalances={hidePortfolioBalances}
+                spotPrice={
+                  spotPriceRegistry &&
+                    !isLoadingSpotPrices &&
+                    !isFetchingSpotPrices
+                      ? getTokenPriceAmountFromRegistry(
+                          spotPriceRegistry,
+                          item.asset
+                        )
+                      : Amount.empty()
```

Comment:
this will always cause a ren-render due to a new `Amount` object instance being created.